gateway-ha/gateway-ha-config.yml

```diff
@@ -26,9 +26,11 @@ server:
   applicationConnectors:
     - type: http
       port: 8090
+      useForwardedHeaders: true
```

Comment:
Please capitalize the commit title. https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages

Reply:
Fixed!